Automattic/jetpack
# WooCommerce Analytics: Queue `add_to_cart` Events

Fixes #9049 

#### Changes proposed in this Pull Request:

* move to queueing all `add_to_cart` events and logging on the next page
* prevent duplicate `product_view` events from product page

#### Testing instructions:

* click any add to cart button and look for event to be logged in the next page load
* add a link to a post or page with href like `http://exampleshop.com/?add-to-cart=${product_id}`
* make sure page view and cart event logged in next page load


#### Proposed changelog entry for your changes:
